src/Compilers/Core/Portable/Emit/InstrumentationKind.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Emit
+{
+    /// <summary>
+    /// Specifies the type of an instrumentation that is performed and be emitted along with the binary output.
+    /// </summary>
+    public enum InstrumentationKind
+    {
+        /// <summary>
+        /// No instrumentation.
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        /// Instruments the binary to add....
```

Comment:
I think the intent was that the name appearing in the code complete the comment. Since this has confused at least one reviewer, I'll make the comment self contained, with apologies to @drognanar .